src/video/emscripten/SDL_emscriptenvideo.c

```diff
@@ -133,13 +139,21 @@ EMSCRIPTEN_KEEPALIVE void Emscripten_SendSystemThemeChangedEvent(void)
 static SDL_VideoDevice *Emscripten_CreateDevice(void)
 {
     SDL_VideoDevice *device;
+    SDL_VideoData *vdata;
```

Comment:
```suggestion
    SDL_VideoData *videodata;
```
Renamed for consistency with other backends